pep-0723.rst

```diff
@@ -0,0 +1,185 @@
+PEP: 723
+Title: Embedding metadata in single-file scripts
+Author: Ofek Lev <ofekmeister@gmail.com>
+PEP-Delegate: Brett Cannon <brett@python.org>
+Discussions-To: https://discuss.python.org/t/29905
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 04-Aug-2023
+Post-History: `19-Jul-2023 <https://discuss.python.org/t/29905>`__
```

Comment:
Does this PEP supersede 722?

https://peps.python.org/pep-0001/#pep-header-preamble

```suggestion
Post-History: `19-Jul-2023 <https://discuss.python.org/t/29905>`__
Replaces: 722
```

Reply:
Also, `Post-History` should point to a dedicated thread for this PEP.